src/Module1/Task1-1.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Write a program which reads a string from the standard input `stdin`,
+ * reverses it and then writes it to the standard output `stdout`.
+ * • The program should be started from `npm script` via nodemon (i.e. npm run task1).
+ * • The program should be running in a stand-by mode and should not be terminated after the first-string processing.
+ *
+ * For example:
+ *   > 12345 678
+ *   >> 876 54321
+ *
+ *   > test data
+ *   >> atad tset
+ *
+ * */
+
+import readline from 'readline';
+
+console.log('\n ********** Please enter your string to reverse **********');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
```

Comment:
If you want to go with readline solution, seams you will need to pass `terminal: false` param also, in other case the  output would be like:

```
123
123    <--- input would be written twice
321
```
`